ext/GDBM_File/GDBM_File.xs

```diff
@@ -145,15 +145,15 @@ output_datum(pTHX_ SV *arg, char *str, int size)
 #define gdbm_setopt(db,optflag,optval,optlen) not_here("gdbm_setopt")
 #endif
 
-#ifndef GDBM_ITEM_NOT_FOUND
-# define GDBM_ITEM_NOT_FOUND GDBM_NO_ERROR
+#if GDBM_VERSION_MAJOR == 1 && GDBM_VERSION_MINOR < 13
+/* Prior to 1.13, only gdbm_fetch set GDBM_ITEM_NOT_FOUND if the requested
```

Comment:
Are there versions earlier than 1.0 that are likely to be in use? E.g. What if someone is using 0.9?...

Reply:
The code and the comment were originally composed by one of the two GNU project maintainers of GDBM (whom I've contacted off-list re this ticket).  In addition, releases with a `0` major version, if they ever existed, are no longer available for download at the [GNU site](http://ftp.gnu.org/gnu/gdbm/).  So I think we're safe here.

Reply:
Corrected typos; corrected $VERSION; squashed commits and force-pushed smoke-me branch.